client/my-sites/post-type-list/index.jsx

```diff
@@ -153,6 +154,7 @@ class PostTypeList extends Component {
 						siteId={ siteId }
 						query={ { ...query, page } } />
 				) ) }
+				{ siteId && <QueryPostTypes siteId={ siteId } /> }
```

Comment:
>Why do you need this one? It seems it would make reusing this less flexible as it'll trigger post types requests on its own.

See extended description of 03ea849.